db/models.py

```diff
@@ -1 +1,29 @@
+from typing import ForwardRef
+
 from django.db import models
+from django.db.models import SET_NULL, CASCADE
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField()
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True, )
+    description = models.TextField(null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=CASCADE)
+    guild = models.ForeignKey(Guild, on_delete=SET_NULL)
```

Comment:
The `on_delete` behavior for the `guild` ForeignKey in the `Player` model is set to `SET_NULL`, which requires `null=True` to be set on the field. Ensure that the `guild` field allows null values.